icu4j/main/classes/core/src/com/ibm/icu/impl/units/MeasureUnitImpl.java

```diff
@@ -266,6 +262,11 @@ public void serialize() {
         this.identifier = result.toString();
     }
 
+    @Override
+    public String toString() {
+        return "MeasureUnitImpl [" + build().getIdentifier() + "]";
+    }
+
```

Comment:
Optional: because this is an urgent bug fix, I would not do any formatting changes.

More verbosely - my suggestions for future PRs (if I wrote this Pull Request, I would do this):

- have one commit that does reformatting only, an one commit that does the logic changes. The reviewer then does not have to figure out what moved code might have changed, and what didn't.
- I would perhaps even do one PR that just does reformatting (no logic changes), perhaps on all related files, then follow that with a separate PR with logic changes. But this makes sense only if the second PR isn't urgent.
- For this urgent PR, after ICU has already been tagged with a release candidate, I would do only logic changes, no formatting changes. This is an "urgent burden" on the reviewers, when we want to make minimal bugfix changes to the Release Candidate.

I can't review formatting changes yet, since I'm not yet familiar with ICU4J style (and haven't started using Eclipse).

Reply:
Actually, prior to this PR, I had touched only toString and three more lines. Yoshito touched the import statements in b2edfb89ccb534ff882954328c0d65d6721bd688 - fixing them according to Eclipse-configured style? This PR is *undoing* those changes again, so clearly IntelliJ is *not* following Eclipse style.

@younies please undo the formatting changes that aren't related to the logic changes. Having warring IDEs will be bad for every PR (next time Eclipse formats this, these changes get undone again).

Reply:
For easier review, this diff is the same code as commit 6a15cad, but shuffled around to make it minimal:
https://github.com/unicode-org/icu/compare/master...icu-units:minimal_diff?expand=1

Reply:
(FYI: pushed to this branch/PR.)